### ISSUE TITLE
How to install the module com.kdab.dockwidgets?

### ISSUE THREAD

**Issue by winnerLu** [KDAB/KDDockWidgets#428]
![acc683c2525150d76272f78b9fdddc6](https://github.com/KDAB/KDDockWidgets/assets/87656628/b39f295f-4240-4870-adcb-324d70879697)

**Comment by iamsergio** [comment]
Do you get an actual build error, or is that just QtCreator code model nagging ?

**Comment by winnerLu** [comment]
I use the example build with the same error, but the build is fine and the program works fine. But when I create a new project, and also import this module, the build gives me an error saying that the module is not found.

**Comment by iamsergio** [comment]
That's some warning in the QtCreator QML code model.

It's out of scope for KDDW to fix and I don't use QtCreator. 
See for example https://forum.qt.io/topic/133555/qml-module-not-found-but-app-is-running/5 for clues.

The warning is benign though, and should not impact the usage.

**Comment by winnerLu** [comment]
> Ok. Can you upload your project to GitHub so I can have a look and fix ? I don't know what's causing it for you, so I need to have a look.

Thank you very much. I've solved it. Needed to Initializes the desired frontend in main.cpp. But since the compiler didn't find the module, there were no code hints as well as auto-completion. It doesn't affect the use, but it would be great to give some clues. I think it has to do with the files QmlTypes.h and QmlTypes.cpp where the module is registered. But I didn't find these two files in the location where I installed KDDW and couldn't bring them in.

Translated with www.DeepL.com/Translator (free version)
